components/panel/body.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
+
+/**
+ * Internal dependencies
+ */
+import PanelBodyToggle from './body-toggle';
+
+class PanelBody extends Component {
+	constructor( props ) {
+		super( ...arguments );
+		this.state = {
+			opened: props.initialOpen === undefined ? true : props.initialOpen,
+		};
+		this.toggle = this.toggle.bind( this );
+	}
+
+	toggle( event ) {
+		event.preventDefault();
```

Comment:
Ah, I guess this would prevent the form submission anyways. Still might be good to be intentional with the `type` prop on `button`.